### ISSUE TITLE
Migrate:refresh broken

### ISSUE THREAD

**Issue by foxted** [cachethq/Cachet#280]
I noticed this issue during the translation changes: it is impossible when the database has been filled to do an `artisan migrate:refresh` command. Here is the error message: 

```
Missing argument 1 for Illuminate\Database\Schema\Blueprint::dropColumn(), called in /Users/foxted/Sites/cachet/app/database/migrations/2015_01_03_120438_AlterTableUsersAddA
  piKey.php on line 29 and defined
```

The same goes for `artisan migrate:reset`, even with the `--force` option. I didn't want to include it in my PR since it wasn't really related, but I'll be happy to fix it.

**Comment by jbrooksuk** [comment]
I believe the reason this may be the case is the foreign keys?

**Comment by foxted** [comment]
No, in the `2015_01_03_120438_AlterTableUsersAddApiKey.php` file, there is no arguments passed in the `dropColumn` method:

```php
public function down()
{
    Schema::table('users', function (Blueprint $table) {
        $table->dropColumn();
    });
}
```

The fix should be: 

```php
public function down()
{
    Schema::table('users', function (Blueprint $table) {
        $table->dropColumn('api_key');
    });
}
```

**Comment by jbrooksuk** [comment]
:facepunch: :smile:  (my face).

Can you make a PR?

**Comment by foxted** [comment]
Sure. Shouldn't I wait for the lang branch to be merged? (Even if there's no risk to cause any conflicts)

**Comment by jbrooksuk** [comment]
Closed by f4ddad07212636aebb09a947e3106f5a4b257d6a.
